src/service/components/pulseaudio.js

```diff
@@ -265,4 +268,4 @@ const Mixer = GObject.registerClass({
 /**
  * The service class for this component
  */
-var Component = Mixer;
+export default Mixer;
```

Comment:
Which means the default export from the module is `null`.

Which appears to be exactly what we're seeing in the tests:

```text
not ok 7 The telephony plugin can lower and restore the volume when
 the phone is ringing
# Message: Error: <spyOn> : could not find an object to spy upon for
 lowerVolume()\nUsage: spyOn(<object>, <methodName>) in
 file:///usr/libexec/installed-tests/gsconnect/jasmine.js (line 7936)
# Stack:
#   error properties: Object({ matches: Function })
#   @file:///usr/libexec/installed-tests/gsconnect/plugins/testTelephonyPlugin.js:151:18
#   setTimeout/source<@resource:///org/gnome/gjs/modules/esm/_timers.js:72:9
#   _init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34
# Message: TypeError: localMixer is undefined in
 file:///usr/libexec/installed-tests/gsconnect/plugins/testTelephonyPlugin.js (line 162)
# Stack:
#   error properties: Object({ matches: Function })
#   @file:///usr/libexec/installed-tests/gsconnect/plugins/testTelephonyPlugin.js:162:13
#   setTimeout/source<@resource:///org/gnome/gjs/modules/esm/_timers.js:72:9
#   _init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34
```